namespace PerseusPluginLib.Test.Annot
{
    public static class Examples
    {
        public static string MouseUniprotAnnotations =
@"UniProt	GOBP name	GOMF name	GOCC name	GOBP slim name	GOCC slim name	KEGG name	Pfam	GSEA	Keywords	Corum	GOBP	GOMF	GOCC	GOBP slim	GOCC slim	ENSG	ENSP	ENST	Isoform ID	Flybase	Gene name	Interpro	Interpro name	KEGG	MGI	MIM	PDB	Pfam	PRINTS	PRINTS name	Prosite	Prosite name	Protein name	Reactome	SGD	SMART	SMART name	Taxonomy	UniProt names	WormBase	EC	eggNOG
#!{Type}	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Text	Text	Text	Text	Text	Text	Categorical	Categorical	Categorical	Text	Categorical	Text	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Categorical	Text	Text	Text	Text
P34968;B1ATN5;Q5WRU6	behavioral fear response;behavioral defense response;nucleobase-containing compound metabolic process;purine nucleotide metabolic process;purine nucleotide biosynthetic process;cGMP biosynthetic process;nucleoside phosphate metabolic process;nitrogen compound metabolic process;transport;ion transport;cation transport;calcium ion transport;cellular ion homeostasis;cellular calcium ion homeostasis;cellular metal ion homeostasis;response to stress;defense response;signal transduction;cell surface receptor linked signaling pathway;G-protein coupled receptor protein signaling pathway;activation of phospholipase C activity by G-protein coupled receptor protein signaling pathway coupled to IP3 second messenger;activation of phospholipase C activity;elevation of cytosolic calcium ion concentration;activation of phospholipase C activity by serotonin receptor signaling pathway;serotonin receptor signaling pathway;behavior;locomotory behavior;feeding behavior;metabolic process;biosynthetic process;nucleotide metabolic process;nucleoside monophosphate metabolic process;nucleoside monophosphate biosynthetic process;nucleotide biosynthetic process;cyclic nucleotide metabolic process;cyclic nucleotide biosynthetic process;regulation of biosynthetic process;positive regulation of biosynthetic process;positive regulation of metabolic process;regulation of signal transduction;positive regulation of signal transduction;cellular process;regulation of phosphatidylinositol biosynthetic process;positive regulation of phosphatidylinositol biosynthetic process;regulation of phospholipase activity;positive regulation of phospholipase activity;regulation of intracellular protein kinase cascade;regulation of cell communication;positive regulation of cell communication;positive regulation of intracellular protein kinase cascade;positive regulation of phospholipase C activity;heterocycle biosynthetic process;regulation of lipid metabolic process;regulation of metabolic process;cellular homeostasis;second-messenger-mediated signaling;regulation of signaling;positive regulation of signaling;metal ion transport;cellular cation homeostasis;regulation of cellular metabolic process;positive regulation of cellular metabolic process;regulation of cellular biosynthetic process;positive regulation of cellular biosynthetic process;regulation of neurological system process;regulation of appetite;multicellular organismal process;regulation of homeostatic process;regulation of localization;multicellular organismal response to stress;cellular nitrogen compound metabolic process;nucleobase-containing compound biosynthetic process;intracellular signal transduction;response to chemical stimulus;response to drug;homeostatic process;fear response;positive regulation of catalytic activity;regulation of corticotropin-releasing hormone secretion;regulation of MAPKKK cascade;positive regulation of MAPKKK cascade;regulation of system process;positive regulation of molecular function;cellular metabolic process;primary metabolic process;cellular biosynthetic process;cellular nitrogen compound biosynthetic process;small molecule metabolic process;regulation of cell differentiation;positive regulation of cell differentiation;regulation of fat cell differentiation;positive regulation of fat cell differentiation;positive regulation of lipid metabolic process;cGMP metabolic process;heterocycle metabolic process;regulation of hormone secretion;positive regulation of lipid biosynthetic process;regulation of lipid biosynthetic process;inositol phosphate-mediated signaling;positive regulation of biological process;negative regulation of biological process;positive regulation of cellular process;regulation of response to stimulus;positive regulation of response to stimulus;chemical homeostasis;regulation of biological process;regulation of catalytic activity;regulation of developmental process;regulation of cellular process;ion homeostasis;response to stimulus;regulation of secretion;regulation of transport;positive regulation of developmental process;release of sequestered calcium ion into cytosol;establishment of localization;regulation of multicellular organismal process;regulation of sequestering of calcium ion;negative regulation of sequestering of calcium ion;regulation of hydrolase activity;positive regulation of hydrolase activity;cytosolic calcium ion homeostasis;elevation of cytosolic calcium ion concentration involved in G-protein signaling coupled to IP3 second messenger;cellular response to stimulus;metal ion homeostasis;calcium ion homeostasis;cation homeostasis;cellular chemical homeostasis;nucleobase-containing small molecule metabolic process;regulation of lipase activity;positive regulation of lipase activity;regulation of cellular localization;cytosolic calcium ion transport;calcium ion transport into cytosol;biological regulation;regulation of biological quality;regulation of molecular function;regulation of ERK1 and ERK2 cascade;positive regulation of ERK1 and ERK2 cascade;divalent metal ion transport;regulation of phospholipid biosynthetic process;positive regulation of phospholipid biosynthetic process;cellular divalent inorganic cation homeostasis;divalent inorganic cation homeostasis;divalent inorganic cation transport;purine-containing compound metabolic process;purine-containing compound biosynthetic process;regulation of primary metabolic process;regulation of ion homeostasis	5-HT2 receptor activity;signal transducer activity;receptor activity;transmembrane signaling receptor activity;G-protein coupled receptor activity;serotonin receptor activity;binding;drug binding;G-protein coupled amine receptor activity;signaling receptor activity;amine binding;serotonin binding;molecular transducer activity;1-(4-iodo-2,5-dimethoxyphenyl)propan-2-amine binding	plasma membrane;integral to plasma membrane;external side of plasma membrane;membrane;integral to membrane;intrinsic to membrane;intrinsic to plasma membrane;membrane part;plasma membrane part;cell part	nucleobase-containing compound metabolic process;nitrogen compound metabolic process;transport;ion transport;cellular ion homeostasis;response to stress;signal transduction;behavior;metabolic process;biosynthetic process;nucleotide metabolic process;cellular process;cellular homeostasis;multicellular organismal process;cellular nitrogen compound metabolic process;response to chemical stimulus;response to drug;homeostatic process;cellular metabolic process;primary metabolic process;small molecule metabolic process;heterocycle metabolic process;response to stimulus;regulation of transport;establishment of localization;nucleobase-containing small molecule metabolic process;biological regulation	plasma membrane;membrane;cell part	Calcium signaling pathway;Neuroactive ligand-receptor interaction;Gap junction	7tm_1	AACTTT_UNKNOWN;AATGTGA,MIR-23A,MIR-23B;ACTGCCT,MIR-34B;BEHAVIOR;BENPORATH_EED_TARGETS;BENPORATH_ES_WITH_H3K27ME3;BENPORATH_PRC2_TARGETS;BENPORATH_SUZ12_TARGETS;CACTGCC,MIR-34A,MIR-34C,MIR-449;CAGCTG_V$AP4_Q5;CAGGTG_V$E12_Q6;CCCAGAG,MIR-326;CCTGCTG,MIR-214;CELL_CELL_SIGNALING;chrxq24;CTTTGCA,MIR-527;FEEDING_BEHAVIOR;GATTGGY_V$NFY_Q6_01;GCANCTGNY_V$MYOD_Q6;GCATTTG,MIR-105;GCCNNNWTAAR_UNKNOWN;GCM_HMGA2;GGCAGTG,MIR-324-3P;GGGAGGRR_V$MAZ_Q6;GOZGIT_ESR1_TARGETS_UP;GSE15930_STIM_VS_STIM_AND_IFNAB_24H_CD8_T_CELL_UP;GSE15930_STIM_VS_STIM_AND_IFNAB_72H_CD8_T_CELL_UP;GSE15930_STIM_VS_STIM_AND_IL-12_24H_CD8_T_CELL_UP;GSE15930_STIM_VS_STIM_AND_IL-12_72H_CD8_T_CELL_DN;GSE15930_STIM_VS_STIM_AND_TRICHOSTATINA_24H_CD8_T_CELL_DN;GSE15930_STIM_VS_STIM_AND_TRICHOSTATINA_72H_CD8_T_CELL_DN;GSE16522_MEMORY_VS_NAIVE_CD8_TCELL_UP;GSE2706_LPS_VS_R848_AND_LPS_2H_STIM_DC_UP;GSE29618_PRE_VS_DAY7_POST_LAIV_FLU_VACCINE_PDC_UP;GSE360_CTRL_VS_B_MALAYI_HIGH_DOSE_DC_DN;GSE360_L_DONOVANI_VS_L_MAJOR_DC_UP;GSE9650_GP33_VS_GP276_LCMV_SPECIFIC_EXHAUSTED_CD8_TCELL_UP;GSE9650_NAIVE_VS_EXHAUSTED_CD8_TCELL_DN;GTAAGAT,MIR-200A;GTCTACC,MIR-379;HAN_SATB1_TARGETS_DN;HAN_SATB1_TARGETS_UP;KEGG_CALCIUM_SIGNALING_PATHWAY;KEGG_GAP_JUNCTION;KEGG_NEUROACTIVE_LIGAND_RECEPTOR_INTERACTION;LEE_NEURAL_CREST_STEM_CELL_DN;LEIN_MIDBRAIN_MARKERS;LEIN_PONS_MARKERS;MEISSNER_BRAIN_HCP_WITH_H3K4ME3_AND_H3K27ME3;MEISSNER_NPC_HCP_WITH_H3_UNMETHYLATED;MIKKELSEN_IPS_WITH_HCP_H3K27ME3;MIKKELSEN_MEF_HCP_WITH_H3_UNMETHYLATED;MODULE_146;MORF_CAMK4;MORF_CDH4;MORF_MAGEA9;MORF_RAD51L3;MORF_SUPT3H;NEUROLOGICAL_SYSTEM_PROCESS;PTEN_DN.V1_UP;REACTOME_AMINE_LIGAND_BINDING_RECEPTORS;REACTOME_CLASS_A1_RHODOPSIN_LIKE_RECEPTORS;REACTOME_G_ALPHA_Q_SIGNALLING_EVENTS;REACTOME_GASTRIN_CREB_SIGNALLING_PATHWAY_VIA_PKC_AND_MAPK;REACTOME_GPCR_DOWNSTREAM_SIGNALING;REACTOME_GPCR_LIGAND_BINDING;REACTOME_SEROTONIN_RECEPTORS;REACTOME_SIGNALING_BY_GPCR;RYTTCCTG_V$ETS2_B;SYNAPTIC_TRANSMISSION;SYSTEM_PROCESS;TAATTA_V$CHX10_01;TACAATC,MIR-508;TAGCTTT,MIR-9;TAKEDA_TARGETS_OF_NUP98_HOXA9_FUSION_8D_DN;TARTE_PLASMA_CELL_VS_PLASMABLAST_UP;TCATCTC,MIR-143;TGAGATT,MIR-216;TGCCAAR_V$NF1_Q6;TGCCTTA,MIR-124A;TGCTGCT,MIR-15A,MIR-16,MIR-15B,MIR-195,MIR-424,MIR-497;TRANSMISSION_OF_NERVE_IMPULSE;TTANTCA_UNKNOWN;V$AFP1_Q6;V$AREB6_04;V$ATF1_Q6;V$CEBP_Q2;V$CEBP_Q3;V$CEBPA_01;V$CETS1P54_01;V$CHOP_01;V$E2F_Q6_01;V$GCM_Q2;V$GR_01;V$IRF_Q6;V$ISRE_01;V$MYB_Q5_01;V$MYOD_01;V$NFY_Q6;V$NKX25_02;V$PAX4_02;V$PIT1_Q6;V$S8_01;V$TAL1ALPHAE47_01;V$TAL1BETAE47_01;V$TAL1BETAITF2_01;V$TST1_01;WONG_ADULT_TISSUE_STEM_MODULE;YCATTAA_UNKNOWN;YNGTTNNNATT_UNKNOWN;YOSHIMURA_MAPK8_TARGETS_UP;YTAAYNGCT_UNKNOWN	Behavior;Cellmembrane;Completeproteome;Disulfidebond;Glycoprotein;G-proteincoupledreceptor;Membrane;Receptor;Referenceproteome;Signal;Transducer;Transmembrane;Transmembranehelix		GO:0001662;GO:0002209;GO:0006139;GO:0006163;GO:0006164;GO:0006182;GO:0006753;GO:0006807;GO:0006810;GO:0006811;GO:0006812;GO:0006816;GO:0006873;GO:0006874;GO:0006875;GO:0006950;GO:0006952;GO:0007165;GO:0007166;GO:0007186;GO:0007200;GO:0007202;GO:0007204;GO:0007208;GO:0007210;GO:0007610;GO:0007626;GO:0007631;GO:0008152;GO:0009058;GO:0009117;GO:0009123;GO:0009124;GO:0009165;GO:0009187;GO:0009190;GO:0009889;GO:0009891;GO:0009893;GO:0009966;GO:0009967;GO:0009987;GO:0010511;GO:0010513;GO:0010517;GO:0010518;GO:0010627;GO:0010646;GO:0010647;GO:0010740;GO:0010863;GO:0018130;GO:0019216;GO:0019222;GO:0019725;GO:0019932;GO:0023051;GO:0023056;GO:0030001;GO:0030003;GO:0031323;GO:0031325;GO:0031326;GO:0031328;GO:0031644;GO:0032098;GO:0032501;GO:0032844;GO:0032879;GO:0033555;GO:0034641;GO:0034654;GO:0035556;GO:0042221;GO:0042493;GO:0042592;GO:0042596;GO:0043085;GO:0043397;GO:0043408;GO:0043410;GO:0044057;GO:0044093;GO:0044237;GO:0044238;GO:0044249;GO:0044271;GO:0044281;GO:0045595;GO:0045597;GO:0045598;GO:0045600;GO:0045834;GO:0046068;GO:0046483;GO:0046883;GO:0046889;GO:0046890;GO:0048016;GO:0048518;GO:0048519;GO:0048522;GO:0048583;GO:0048584;GO:0048878;GO:0050789;GO:0050790;GO:0050793;GO:0050794;GO:0050801;GO:0050896;GO:0051046;GO:0051049;GO:0051094;GO:0051209;GO:0051234;GO:0051239;GO:0051282;GO:0051283;GO:0051336;GO:0051345;GO:0051480;GO:0051482;GO:0051716;GO:0055065;GO:0055074;GO:0055080;GO:0055082;GO:0055086;GO:0060191;GO:0060193;GO:0060341;GO:0060401;GO:0060402;GO:0065007;GO:0065008;GO:0065009;GO:0070372;GO:0070374;GO:0070838;GO:0071071;GO:0071073;GO:0072503;GO:0072507;GO:0072511;GO:0072521;GO:0072522;GO:0080090;GO:2000021	GO:0001587;GO:0004871;GO:0004872;GO:0004888;GO:0004930;GO:0004993;GO:0005488;GO:0008144;GO:0008227;GO:0038023;GO:0043176;GO:0051378;GO:0060089;GO:0071886	GO:0005886;GO:0005887;GO:0009897;GO:0016020;GO:0016021;GO:0031224;GO:0031226;GO:0044425;GO:0044459;GO:0044464	GO:0006139;GO:0006807;GO:0006810;GO:0006811;GO:0006873;GO:0006950;GO:0007165;GO:0007610;GO:0008152;GO:0009058;GO:0009117;GO:0009987;GO:0019725;GO:0032501;GO:0034641;GO:0042221;GO:0042493;GO:0042592;GO:0044237;GO:0044238;GO:0044281;GO:0046483;GO:0050896;GO:0051049;GO:0051234;GO:0055086;GO:0065007	GO:0005886;GO:0016020;GO:0044464	ENSMUSG00000041380;ENSMUSG00000041380	ENSMUSP00000043936;ENSMUSP00000094021	ENSMUST00000036303;ENSMUST00000096299			Htr2c	IPR000377;IPR002231;IPR000276;IPR017452	5HT2C_rcpt;5HT_rcpt;GPCR_Rhodpsn;GPCR_Rhodpsn_7TM	ko04020;ko04080;ko04540	MGI:96281			PF00001	PR00517;PR01101;PR00237	5HT2CRECEPTR;5HTRECEPTOR;GPCRRHODOPSN	PS00237;PS50262	G_PROTEIN_RECEP_F1_1;G_PROTEIN_RECEP_F1_2	5-hydroxytryptamine receptor 2C	REACT_332615;REACT_347934				Mus musculus	5HT2C_MOUSE			NOG247243
Q91V24;Q9JL36	system process;regulation of nucleotide metabolic process;protein modification process;transport;lipid transport;endocytosis;phagocytosis;plasma membrane organization;signal transduction;cell surface receptor linked signaling pathway;behavior;learning or memory;memory;protein localization;metabolic process;regulation of biosynthetic process;negative regulation of biosynthetic process;negative regulation of metabolic process;regulation of catabolic process;regulation of signal transduction;positive regulation of signal transduction;cellular process;membrane invagination;regulation of macromolecule biosynthetic process;negative regulation of macromolecule biosynthetic process;regulation of glycoprotein biosynthetic process;negative regulation of glycoprotein biosynthetic process;negative regulation of macromolecule metabolic process;regulation of intracellular protein kinase cascade;regulation of cell communication;positive regulation of cell communication;positive regulation of intracellular protein kinase cascade;regulation of cholesterol efflux;positive regulation of cholesterol efflux;phospholipid transport;sterol transport;cellular component organization;cellular membrane organization;vesicle-mediated transport;phospholipid scrambling;peptide cross-linking;regulation of nucleobase-containing compound metabolic process;regulation of metabolic process;protein metabolic process;cellular component assembly;regulation of signaling;positive regulation of signaling;regulation of endocytosis;cholesterol transport;regulation of nucleotide catabolic process;regulation of cellular metabolic process;negative regulation of cellular metabolic process;regulation of cellular biosynthetic process;negative regulation of cellular biosynthetic process;regulation of cellular catabolic process;regulation of lipid transport;positive regulation of lipid transport;regulation of sterol transport;positive regulation of sterol transport;regulation of cholesterol transport;positive regulation of cholesterol transport;multicellular organismal process;negative regulation of ATPase activity;positive regulation of ATPase activity;regulation of localization;macromolecule localization;regulation of purine nucleotide catabolic process;cholesterol efflux;protein localization to organelle;phospholipid efflux;plasma lipoprotein particle assembly;high-density lipoprotein particle assembly;protein localization to nucleus;cellular protein localization;apolipoprotein A-I-mediated signaling pathway;regulation of amyloid precursor protein biosynthetic process;negative regulation of amyloid precursor protein biosynthetic process;positive regulation of catalytic activity;negative regulation of catalytic activity;macromolecule metabolic process;regulation of MAPKKK cascade;positive regulation of MAPKKK cascade;macromolecule modification;regulation of ATPase activity;macromolecular complex subunit organization;negative regulation of molecular function;positive regulation of molecular function;cellular metabolic process;primary metabolic process;cellular macromolecule metabolic process;cellular protein metabolic process;positive regulation of endocytosis;positive regulation of biological process;negative regulation of biological process;positive regulation of cellular process;negative regulation of cellular process;regulation of response to stimulus;positive regulation of response to stimulus;regulation of phagocytosis;positive regulation of phagocytosis;regulation of biological process;regulation of catalytic activity;regulation of cellular process;neurological system process;cognition;response to stimulus;regulation of transport;positive regulation of transport;regulation of cellular component organization;positive regulation of cellular component organization;regulation of nitrogen compound metabolic process;localization;establishment of localization;regulation of hydrolase activity;positive regulation of hydrolase activity;negative regulation of hydrolase activity;cellular localization;cellular response to stimulus;transmembrane transport;regulation of macromolecule metabolic process;regulation of vesicle-mediated transport;membrane organization;macromolecular complex assembly;protein-lipid complex assembly;biological regulation;regulation of biological quality;regulation of molecular function;regulation of ERK1 and ERK2 cascade;positive regulation of ERK1 and ERK2 cascade;cellular macromolecule localization;organic substance transport;protein-lipid complex subunit organization;plasma lipoprotein particle organization;cellular component organization or biogenesis;regulation of primary metabolic process;regulation of membrane lipid distribution;regulation of cellular macromolecule biosynthetic process;negative regulation of cellular macromolecule biosynthetic process	nucleotide binding;catalytic activity;signal transducer activity;receptor activity;transmembrane signaling receptor activity;transporter activity;lipid transporter activity;binding;protein binding;ATP binding;phospholipid transporter activity;sterol transporter activity;primary active transmembrane transporter activity;P-P-bond-hydrolysis-driven transmembrane transporter activity;pyrophosphatase activity;hydrolase activity;hydrolase activity, acting on acid anhydrides;hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides;hydrolase activity, acting on acid anhydrides, catalyzing transmembrane movement of substances;ATPase activity;purine nucleotide binding;nucleoside-triphosphatase activity;cholesterol transporter activity;active transmembrane transporter activity;transmembrane transporter activity;substrate-specific transporter activity;apolipoprotein receptor activity;adenyl nucleotide binding;ribonucleotide binding;purine ribonucleotide binding;adenyl ribonucleotide binding;apolipoprotein binding;apolipoprotein A-I binding;apolipoprotein A-I receptor activity;purine ribonucleoside triphosphate binding;signaling receptor activity;ATPase activity, coupled;ATPase activity, coupled to transmembrane movement of substances;ATPase activity, coupled to movement of substances;molecular transducer activity	Golgi membrane;phagocytic cup;intracellular;cytoplasm;Golgi apparatus;plasma membrane;integral to plasma membrane;cell surface;endosome membrane;membrane;integral to membrane;apical plasma membrane;organelle membrane;intrinsic to membrane;intrinsic to plasma membrane;cell projection membrane;leading edge membrane;ruffle membrane;organelle;membrane-bounded organelle;intracellular organelle;intracellular membrane-bounded organelle;organelle part;intracellular part;membrane part;Golgi apparatus part;endosomal part;cytoplasmic part;intracellular organelle part;plasma membrane part;cell projection part;cell part	protein modification process;transport;lipid transport;endocytosis;plasma membrane organization;signal transduction;behavior;metabolic process;cellular process;membrane invagination;cellular component organization;cellular membrane organization;vesicle-mediated transport;protein metabolic process;cellular component assembly;multicellular organismal process;macromolecule metabolic process;macromolecule modification;cellular metabolic process;primary metabolic process;neurological system process;response to stimulus;regulation of transport;localization;establishment of localization;cellular localization;transmembrane transport;membrane organization;macromolecular complex assembly;biological regulation	intracellular;cytoplasm;Golgi apparatus;plasma membrane;cell surface;membrane;organelle;intracellular organelle;intracellular membrane-bounded organelle;cell part	ABC transporters	ABC_tran	ADENYL_NUCLEOTIDE_BINDING;ADENYL_RIBONUCLEOTIDE_BINDING;AKT_UP.V1_UP;AKT_UP_MTOR_DN.V1_UP;ATP_BINDING;BENPORATH_CYCLING_GENES;CAIRO_LIVER_DEVELOPMENT_DN;chr19p13;CTTTAAR_UNKNOWN;ESC_V6.5_UP_EARLY.V1_DN;FEVR_CTNNB1_TARGETS_UP;GATTGGY_V$NFY_Q6_01;GINESTIER_BREAST_CANCER_ZNF217_AMPLIFIED_DN;GOBERT_OLIGODENDROCYTE_DIFFERENTIATION_DN;GSE11924_TH2_VS_TH17_CD4_TCELL_DN;GSE12845_IGD_POS_BLOOD_VS_NAIVE_TONSIL_BCELL_UP;GSE14308_TH17_VS_NAIVE_CD4_TCELL_DN;GSE15324_NAIVE_VS_ACTIVATED_ELF4_KO_CD8_TCELL_UP;GSE17721_0.5H_VS_8H_GARDIQUIMOD_BMDM_DN;GSE17721_POLYIC_VS_CPG_16H_BMDM_DN;GSE20715_WT_VS_TLR4_KO_24H_OZONE_LUNG_DN;GSE20715_WT_VS_TLR4_KO_48H_OZONE_LUNG_DN;GSE20715_WT_VS_TLR4_KO_LUNG_DN;GSE22886_IGA_VS_IGM_MEMORY_BCELL_DN;GSE22886_IGM_MEMORY_BCELL_VS_BM_PLASMA_CELL_UP;GSE22886_NAIVE_CD4_TCELL_VS_48H_ACT_TH1_UP;GSE24634_NAIVE_CD4_TCELL_VS_DAY5_IL4_CONV_TREG_UP;GSE24634_NAIVE_CD4_TCELL_VS_DAY7_IL4_CONV_TREG_UP;GSE24634_TEFF_VS_TCONV_DAY5_IN_CULTURE_DN;GSE26669_CTRL_VS_COSTIM_BLOCK_MLR_CD8_TCELL_DN;GSE27786_BCELL_VS_NKTCELL_DN;GSE27786_CD8_TCELL_VS_NKTCELL_DN;GSE27786_LIN_NEG_VS_NEUTROPHIL_DN;GSE27786_LIN_NEG_VS_NKCELL_DN;GSE27786_LSK_VS_NKCELL_DN;GSE29618_PRE_VS_DAY7_POST_LAIV_FLU_VACCINE_MONOCYTE_DN;GSE3982_EOSINOPHIL_VS_MAST_CELL_UP;GSE3982_MAST_CELL_VS_BCELL_DN;GSE3982_MEMORY_CD4_TCELL_VS_TH2_UP;GSE6269_HEALTHY_VS_E_COLI_INF_PBMC_UP;INTEGRAL_TO_MEMBRANE;INTRINSIC_TO_MEMBRANE;KEGG_ABC_TRANSPORTERS;LIM_MAMMARY_LUMINAL_MATURE_UP;MACROMOLECULAR_COMPLEX;MAGRANGEAS_MULTIPLE_MYELOMA_IGG_VS_IGA_UP;MARSON_BOUND_BY_FOXP3_UNSTIMULATED;MEMBRANE;MEMBRANE_PART;MILI_PSEUDOPODIA_CHEMOTAXIS_DN;MODULE_256;MODULE_257;MODULE_333;MODULE_427;MODULE_480;MULLIGHAN_MLL_SIGNATURE_1_UP;MULLIGHAN_MLL_SIGNATURE_2_UP;NUCLEOTIDE_BINDING;PANGAS_TUMOR_SUPPRESSION_BY_SMAD1_AND_SMAD5_DN;PEDRIOLI_MIR31_TARGETS_DN;PEREZ_TP53_TARGETS;PROTEIN_COMPLEX;PURINE_NUCLEOTIDE_BINDING;PURINE_RIBONUCLEOTIDE_BINDING;REACTOME_ABC_FAMILY_PROTEINS_MEDIATED_TRANSPORT;REACTOME_ABCA_TRANSPORTERS_IN_LIPID_HOMEOSTASIS;REACTOME_TRANSMEMBRANE_TRANSPORT_OF_SMALL_MOLECULES;ROME_INSULIN_TARGETS_IN_MUSCLE_DN;ROSS_LEUKEMIA_WITH_MLL_FUSIONS;TGANNYRGCA_V$TCF11MAFG_01;V$ALPHACP1_01;V$NFY_01;V$NFY_C;V$NFY_Q6;WANG_RESPONSE_TO_GSK3_INHIBITOR_SB216763_UP;WHITFIELD_CELL_CYCLE_G1_S;WTTGKCTG_UNKNOWN	ATP-binding;Cellmembrane;Completeproteome;Disulfidebond;Endosome;Glycoprotein;Golgiapparatus;Membrane;Nucleotide-binding;Phagocytosis;Referenceproteome;Repeat;Transmembrane;Transmembranehelix;Transport		GO:0003008;GO:0006140;GO:0006464;GO:0006810;GO:0006869;GO:0006897;GO:0006909;GO:0007009;GO:0007165;GO:0007166;GO:0007610;GO:0007611;GO:0007613;GO:0008104;GO:0008152;GO:0009889;GO:0009890;GO:0009892;GO:0009894;GO:0009966;GO:0009967;GO:0009987;GO:0010324;GO:0010556;GO:0010558;GO:0010559;GO:0010561;GO:0010605;GO:0010627;GO:0010646;GO:0010647;GO:0010740;GO:0010874;GO:0010875;GO:0015914;GO:0015918;GO:0016043;GO:0016044;GO:0016192;GO:0017121;GO:0018149;GO:0019219;GO:0019222;GO:0019538;GO:0022607;GO:0023051;GO:0023056;GO:0030100;GO:0030301;GO:0030811;GO:0031323;GO:0031324;GO:0031326;GO:0031327;GO:0031329;GO:0032368;GO:0032370;GO:0032371;GO:0032373;GO:0032374;GO:0032376;GO:0032501;GO:0032780;GO:0032781;GO:0032879;GO:0033036;GO:0033121;GO:0033344;GO:0033365;GO:0033700;GO:0034377;GO:0034380;GO:0034504;GO:0034613;GO:0038027;GO:0042984;GO:0042985;GO:0043085;GO:0043086;GO:0043170;GO:0043408;GO:0043410;GO:0043412;GO:0043462;GO:0043933;GO:0044092;GO:0044093;GO:0044237;GO:0044238;GO:0044260;GO:0044267;GO:0045807;GO:0048518;GO:0048519;GO:0048522;GO:0048523;GO:0048583;GO:0048584;GO:0050764;GO:0050766;GO:0050789;GO:0050790;GO:0050794;GO:0050877;GO:0050890;GO:0050896;GO:0051049;GO:0051050;GO:0051128;GO:0051130;GO:0051171;GO:0051179;GO:0051234;GO:0051336;GO:0051345;GO:0051346;GO:0051641;GO:0051716;GO:0055085;GO:0060255;GO:0060627;GO:0061024;GO:0065003;GO:0065005;GO:0065007;GO:0065008;GO:0065009;GO:0070372;GO:0070374;GO:0070727;GO:0071702;GO:0071825;GO:0071827;GO:0071840;GO:0080090;GO:0097035;GO:2000112;GO:2000113	GO:0000166;GO:0003824;GO:0004871;GO:0004872;GO:0004888;GO:0005215;GO:0005319;GO:0005488;GO:0005515;GO:0005524;GO:0005548;GO:0015248;GO:0015399;GO:0015405;GO:0016462;GO:0016787;GO:0016817;GO:0016818;GO:0016820;GO:0016887;GO:0017076;GO:0017111;GO:0017127;GO:0022804;GO:0022857;GO:0022892;GO:0030226;GO:0030554;GO:0032553;GO:0032555;GO:0032559;GO:0034185;GO:0034186;GO:0034188;GO:0035639;GO:0038023;GO:0042623;GO:0042626;GO:0043492;GO:0060089	GO:0000139;GO:0001891;GO:0005622;GO:0005737;GO:0005794;GO:0005886;GO:0005887;GO:0009986;GO:0010008;GO:0016020;GO:0016021;GO:0016324;GO:0031090;GO:0031224;GO:0031226;GO:0031253;GO:0031256;GO:0032587;GO:0043226;GO:0043227;GO:0043229;GO:0043231;GO:0044422;GO:0044424;GO:0044425;GO:0044431;GO:0044440;GO:0044444;GO:0044446;GO:0044459;GO:0044463;GO:0044464	GO:0006464;GO:0006810;GO:0006869;GO:0006897;GO:0007009;GO:0007165;GO:0007610;GO:0008152;GO:0009987;GO:0010324;GO:0016043;GO:0016044;GO:0016192;GO:0019538;GO:0022607;GO:0032501;GO:0043170;GO:0043412;GO:0044237;GO:0044238;GO:0050877;GO:0050896;GO:0051049;GO:0051179;GO:0051234;GO:0051641;GO:0055085;GO:0061024;GO:0065003;GO:0065007	GO:0005622;GO:0005737;GO:0005794;GO:0005886;GO:0009986;GO:0016020;GO:0043226;GO:0043229;GO:0043231;GO:0044464	ENSMUSG00000035722;ENSMUSG00000035722	ENSMUSP00000043090;ENSMUSP00000115111	ENSMUST00000043866;ENSMUST00000132517			Abca7	IPR003593;IPR026082;IPR003439;IPR017871;IPR030369;IPR027417	AAA+_ATPase;ABC_A;ABC_transporter-like;ABC_transporter_CS;ABCA7;P-loop_NTPase	ko02010	MGI:1351646			PF00005			PS00211;PS50893	ABC_TRANSPORTER_1;ABC_TRANSPORTER_2	ATP-binding cassette sub-family A member 7	REACT_346149		SM00382	AAA	Mus musculus	ABCA7_MOUSE			COG1131
Q9DBM0	system process;transport;lipid transport;response to nutrient;excretion;metabolic process;response to external stimulus;response to extracellular stimulus;negative regulation of intestinal phytosterol absorption;phospholipid transport;sterol transport;digestive system process;intestinal cholesterol absorption;regulation of intestinal cholesterol absorption;cholesterol transport;response to nutrient levels;regulation of lipid transport;negative regulation of lipid transport;regulation of sterol transport;negative regulation of sterol transport;regulation of cholesterol transport;negative regulation of cholesterol transport;multicellular organismal process;regulation of localization;cholesterol efflux;response to chemical stimulus;response to drug;homeostatic process;cholesterol homeostasis;regulation of system process;regulation of digestive system process;negative regulation of intestinal cholesterol absorption;secretion;negative regulation of biological process;chemical homeostasis;regulation of biological process;intestinal absorption;response to stimulus;regulation of transport;negative regulation of transport;establishment of localization;regulation of multicellular organismal process;negative regulation of multicellular organismal process;lipid homeostasis;sterol homeostasis;negative regulation of digestive system process;biological regulation;regulation of biological quality;organic substance transport	nucleotide binding;catalytic activity;transporter activity;lipid transporter activity;binding;protein binding;ATP binding;sterol transporter activity;pyrophosphatase activity;hydrolase activity;hydrolase activity, acting on acid anhydrides;hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides;ATPase activity;purine nucleotide binding;nucleoside-triphosphatase activity;cholesterol transporter activity;substrate-specific transporter activity;adenyl nucleotide binding;ribonucleotide binding;purine ribonucleotide binding;adenyl ribonucleotide binding;purine ribonucleoside triphosphate binding;protein heterodimerization activity;protein dimerization activity	apical plasma membrane;macromolecular complex;ATP-binding cassette (ABC) transporter complex;protein complex;receptor complex;membrane part;plasma membrane part;cell part	transport;lipid transport;excretion;metabolic process;response to external stimulus;response to extracellular stimulus;multicellular organismal process;response to chemical stimulus;response to drug;homeostatic process;secretion;response to stimulus;regulation of transport;establishment of localization;biological regulation	macromolecular complex;ATP-binding cassette (ABC) transporter complex;protein complex;receptor complex;cell part	ABC transporters;Fat digestion and absorption;Bile secretion	ABC2_membrane;ABC_tran	AAAGACA,MIR-511;BRUINS_UVC_RESPONSE_LATE;CHIANG_LIVER_CANCER_SUBCLASS_PROLIFERATION_DN;chr2p21;GSE12366_PLASMA_CELL_VS_MEMORY_BCELL_UP;GSE14769_UNSTIM_VS_240MIN_LPS_BMDM_DN;GSE17721_0.5H_VS_8H_LPS_BMDM_UP;GSE17721_CTRL_VS_LPS_8H_BMDM_DN;GSE17721_CTRL_VS_PAM3CSK4_1H_BMDM_UP;GSE17721_CTRL_VS_PAM3CSK4_2H_BMDM_UP;GSE17721_PAM3CSK4_VS_CPG_2H_BMDM_DN;GUO_TARGETS_OF_IRS1_AND_IRS2;KEGG_ABC_TRANSPORTERS;LEE_LIVER_CANCER_SURVIVAL_UP;MIKKELSEN_MEF_LCP_WITH_H3K27ME3;PROTEIN_DIMERIZATION_ACTIVITY;PROTEIN_HETERODIMERIZATION_ACTIVITY;REACTOME_ABC_FAMILY_PROTEINS_MEDIATED_TRANSPORT;REACTOME_ABCA_TRANSPORTERS_IN_LIPID_HOMEOSTASIS;REACTOME_LIPID_DIGESTION_MOBILIZATION_AND_TRANSPORT;REACTOME_METABOLISM_OF_LIPIDS_AND_LIPOPROTEINS;REACTOME_TRANSMEMBRANE_TRANSPORT_OF_SMALL_MOLECULES;SUMI_HNF4A_TARGETS;V$AP3_Q6;V$GATA1_04;V$GATA3_01;V$GR_Q6_01;V$LMO2COM_02	Alternativesplicing;Completeproteome;Glycoprotein;Membrane;Referenceproteome;Transmembrane;Transmembranehelix;Transport	Abcg5-Abcg8 complex;Abcg8-Abcg4 complex	GO:0003008;GO:0006810;GO:0006869;GO:0007584;GO:0007588;GO:0008152;GO:0009605;GO:0009991;GO:0010949;GO:0015914;GO:0015918;GO:0022600;GO:0030299;GO:0030300;GO:0030301;GO:0031667;GO:0032368;GO:0032369;GO:0032371;GO:0032372;GO:0032374;GO:0032375;GO:0032501;GO:0032879;GO:0033344;GO:0042221;GO:0042493;GO:0042592;GO:0042632;GO:0044057;GO:0044058;GO:0045796;GO:0046903;GO:0048519;GO:0048878;GO:0050789;GO:0050892;GO:0050896;GO:0051049;GO:0051051;GO:0051234;GO:0051239;GO:0051241;GO:0055088;GO:0055092;GO:0060457;GO:0065007;GO:0065008;GO:0071702	GO:0000166;GO:0003824;GO:0005215;GO:0005319;GO:0005488;GO:0005515;GO:0005524;GO:0015248;GO:0016462;GO:0016787;GO:0016817;GO:0016818;GO:0016887;GO:0017076;GO:0017111;GO:0017127;GO:0022892;GO:0030554;GO:0032553;GO:0032555;GO:0032559;GO:0035639;GO:0046982;GO:0046983	GO:0016324;GO:0032991;GO:0043190;GO:0043234;GO:0043235;GO:0044425;GO:0044459;GO:0044464	GO:0006810;GO:0006869;GO:0007588;GO:0008152;GO:0009605;GO:0009991;GO:0032501;GO:0042221;GO:0042493;GO:0042592;GO:0046903;GO:0050896;GO:0051049;GO:0051234;GO:0065007	GO:0032991;GO:0043190;GO:0043234;GO:0043235;GO:0044464	ENSMUSG00000024254	ENSMUSP00000035246	ENSMUST00000045714	Q9DBM0-1		Abcg8	IPR013525;IPR003439;IPR017871;IPR027417	ABC_2_trans;ABC_transporter-like;ABC_transporter_CS;P-loop_NTPase	ko02010;ko04975;ko04976	MGI:1914720			PF01061;PF00005			PS00211;PS50893	ABC_TRANSPORTER_1;ABC_TRANSPORTER_2	ATP-binding cassette sub-family G member 8	REACT_343032;REACT_346149				Mus musculus	ABCG8_MOUSE			COG1131
Q6PE15;Q3TMB4;Q8C3S8;Q8C724;Q8K188	carbohydrate metabolic process;metabolic process;catabolic process;cellular process;carbohydrate catabolic process;glucuronoside metabolic process;glucuronoside catabolic process;cellular metabolic process;primary metabolic process;cellular carbohydrate metabolic process;cellular carbohydrate catabolic process	catalytic activity;hydrolase activity, hydrolyzing O-glycosyl compounds;hydrolase activity;hydrolase activity, acting on glycosyl bonds	mitochondrion;cytosol;organelle;membrane-bounded organelle;intracellular organelle;intracellular membrane-bounded organelle;intracellular part;cytoplasmic part;cell part	carbohydrate metabolic process;metabolic process;catabolic process;cellular process;cellular metabolic process;primary metabolic process	mitochondrion;cytosol;organelle;intracellular organelle;intracellular membrane-bounded organelle;cell part		Abhydrolase_6	ACEVEDO_LIVER_CANCER_DN;ACEVEDO_NORMAL_TISSUE_ADJACENT_TO_LIVER_TUMOR_DN;BENPORATH_CYCLING_GENES;chr3q13;GARGALOVIC_RESPONSE_TO_OXIDIZED_PHOSPHOLIPIDS_BLACK_DN;GOBERT_OLIGODENDROCYTE_DIFFERENTIATION_DN;GSE12845_IGD_POS_BLOOD_VS_NAIVE_TONSIL_BCELL_UP;GSE12845_IGD_POS_BLOOD_VS_PRE_GC_TONSIL_BCELL_UP;GSE13485_DAY1_VS_DAY21_YF17D_VACCINE_PBMC_DN;GSE14000_UNSTIM_VS_16H_LPS_DC_UP;GSE1460_INTRATHYMIC_T_PROGENITOR_VS_NAIVE_CD4_TCELL_CORD_BLOOD_UP;GSE1460_INTRATHYMIC_T_PROGENITOR_VS_THYMIC_STROMAL_CELL_UP;GSE15324_ELF4_KO_VS_WT_ACTIVATED_CD8_TCELL_DN;GSE15324_NAIVE_VS_ACTIVATED_ELF4_KO_CD8_TCELL_UP;GSE17721_0.5H_VS_8H_POLYIC_BMDM_DN;GSE17721_CTRL_VS_CPG_24H_BMDM_UP;GSE17721_CTRL_VS_GARDIQUIMOD_2H_BMDM_UP;GSE17721_CTRL_VS_PAM3CSK4_2H_BMDM_UP;GSE17721_CTRL_VS_POLYIC_0.5H_BMDM_UP;GSE17721_LPS_VS_PAM3CSK4_2H_BMDM_UP;GSE17721_LPS_VS_POLYIC_8H_BMDM_DN;GSE17721_POLYIC_VS_GARDIQUIMOD_2H_BMDM_DN;GSE17721_POLYIC_VS_PAM3CSK4_8H_BMDM_UP;GSE17974_1.5H_VS_72H_IL4_AND_ANTI_IL12_ACT_CD4_TCELL_DN;GSE17974_2.5H_VS_72H_IL4_AND_ANTI_IL12_ACT_CD4_TCELL_DN;GSE22886_DAY1_VS_DAY7_MONOCYTE_IN_CULTURE_DN;GSE22886_UNSTIM_VS_STIM_MEMORY_TCELL_DN;GSE2706_UNSTIM_VS_2H_R848_DC_UP;GSE28237_EARLY_VS_LATE_GC_BCELL_UP;GSE29618_PRE_VS_DAY7_FLU_VACCINE_MONOCYTE_DN;GSE36476_YOUNG_VS_OLD_DONOR_MEMORY_CD4_TCELL_16H_TSST_ACT_UP;GSE37416_CTRL_VS_24H_F_TULARENSIS_LVS_NEUTROPHIL_UP;GSE37416_CTRL_VS_48H_F_TULARENSIS_LVS_NEUTROPHIL_UP;GSE3982_CTRL_VS_PMA_STIM_EOSINOPHIL_UP;GSE7852_LN_VS_FAT_TREG_UP;HOLLMANN_APOPTOSIS_VIA_CD40_DN;KIM_WT1_TARGETS_UP;KRAS.DF.V1_UP;LEE_BMP2_TARGETS_DN;MARSON_BOUND_BY_E2F4_UNSTIMULATED;MULLIGHAN_MLL_SIGNATURE_1_DN;MULLIGHAN_MLL_SIGNATURE_2_DN;PILON_KLF1_TARGETS_DN;RCGCANGCGY_V$NRF1_Q6;VECCHI_GASTRIC_CANCER_EARLY_UP;WHITFIELD_CELL_CYCLE_S;ZHANG_RESPONSE_TO_IKK_INHIBITOR_AND_TNF_DN;ZHOU_INFLAMMATORY_RESPONSE_FIMA_DN;ZHOU_INFLAMMATORY_RESPONSE_LIVE_DN;ZWANG_TRANSIENTLY_UP_BY_1ST_EGF_PULSE_ONLY	Alternativesplicing;Completeproteome;Hydrolase;Mitochondrion;Referenceproteome;Transitpeptide		GO:0005975;GO:0008152;GO:0009056;GO:0009987;GO:0016052;GO:0019389;GO:0019391;GO:0044237;GO:0044238;GO:0044262;GO:0044275	GO:0003824;GO:0004553;GO:0016787;GO:0016798	GO:0005739;GO:0005829;GO:0043226;GO:0043227;GO:0043229;GO:0043231;GO:0044424;GO:0044444;GO:0044464	GO:0005975;GO:0008152;GO:0009056;GO:0009987;GO:0044237;GO:0044238	GO:0005739;GO:0005829;GO:0043226;GO:0043229;GO:0043231;GO:0044464	ENSMUSG00000033157	ENSMUSP00000065282	ENSMUST00000066983	Q6PE15-1		Abhd10	IPR029058;IPR000073	AB_hydrolase;AB_hydrolase_1		MGI:2442422			PF12697					Mycophenolic acid acyl-glucuronide esterase, mitochondrial					Mus musculus	ABHDA_MOUSE		3.1.1.93	COG0596";

    }
}